inc/customizer/colors.php

```diff
@@ -470,58 +590,120 @@ public function __construct() {
 		 */
 		$default_scheme = array(
 			'default' => array(
-				'label'  => esc_html__( 'Default', 'primer' ),
-				'colors' => array_combine(
+				'label'      => esc_html__( 'Default', 'primer' ),
+				'colors'     => array_combine(
 					array_keys( $this->colors ),
 					wp_list_pluck( $this->colors, 'default' )
 				),
+				'primary'    => sanitize_hex_color( '#' . get_theme_mod( 'primary_color', $this->get_default_color( 'primary_color', 'default' ) ) ),
+				'secondary'  => sanitize_hex_color( '#' . get_theme_mod( 'secondary_color', $this->get_default_color( 'secondary_color', 'default' ) ) ),
+				'tertiary'   => sanitize_hex_color( '#' . get_theme_mod( 'tertiary_color', $this->get_default_color( 'tertiary_color', 'default' ) ) ),
+				'quaternary' => sanitize_hex_color( '#' . get_theme_mod( 'quaternary_color', $this->get_default_color( 'quaternary_color', 'default' ) ) ),
+				'quinary'    => sanitize_hex_color( '#' . get_theme_mod( 'quinary_color', $this->get_default_color( 'quinary_color', 'default' ) ) ),
```

Comment:
It looks like this section is causing the errors that I mentioned in slack this morning. `$this->color_schemes` looks like it's empty in `$this->get_default_color` run in the construction of `colors.php`